bin/all_sky_search/pycbc_calculate_dq

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+""" Ingest and process DQ data
+"""
+import logging, argparse, numpy, h5py, time, copy
+import pycbc, pycbc.strain
+from pycbc.version import git_verbose_msg as version
+from pycbc.workflow import resolve_td_option
+from ligo.segments import segmentlist, segment
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--version', action='version', version=version)
+parser.add_argument('--verbose', action="store_true")
+parser.add_argument("--dq-sample-rate", default=1, type=int,
+                    help="Sample rate to calculate dq timeseries, "
+                         "in Hz [default=1Hz]")
+parser.add_argument("--dq-sample-method", required=True,
+                    help="Method used to calculate dq timeseries. "
+                         "Options are 'max' or 'blrms'")
+parser.add_argument("--gps-start-time", type=int,required=True,
+                    help="The gps start time of the data "
+                          "(integer seconds)")
+parser.add_argument("--gps-end-time", type=int,required=True,
+                    help="The gps end time of the data "
+                         "(integer seconds)")
+parser.add_argument("--output-file", required=True,
+                    help="name of output file")
+
+pycbc.strain.insert_strain_option_group(parser, gps_times=False)
+
+parser.set_defaults(pad_data=0)
+
+args = parser.parse_args()
+pycbc.init_logging(args.verbose)
+
+def get_dq(seg):
+    """ Get the DQ data for the given data chunck
+    """
+    
+    logging.info('Getting dq values for %.1f-%.1f (%.1f s)', seg[0],
+                 seg[1], abs(seg[1]-seg[0]))
+    argstmp = copy.deepcopy(args)
+    argstmp.gps_start_time = int(seg[0])
+    argstmp.gps_end_time = int(seg[1])
+    tmp_segment = segment([argstmp.gps_start_time, argstmp.gps_end_time])
+    argstmp.channel_name = resolve_td_option(args.channel_name, tmp_segment)
+
+    try:
+        gwstrain = pycbc.strain.from_cli(argstmp)
```

Comment:
If I understand correctly the purpose of this script, the idea is to use the `pycbc.strain.from_cli()` infrastructure to actually read a DQ time series (as opposed to h(t) strain data). In that case, `gwstrain` is probably not the best variable name here. Would `dq_data` be more appropriate?